.circleci/config.yml

```diff
@@ -4,12 +4,19 @@ commands:
     prep:
       steps:
         - checkout
+        - run:
+            name: Update
+            command: sudo apt update
+        - run:
+            name: Install NPM
+            command: sudo apt install nodejs npm
         - run:
             name: Install python3 and pip3
-            command: sudo apt-get update && sudo apt-get -y install python3 python3-pip
+            command: sudo apt-get update && sudo apt-get -y install python3 python3-pip && sudo npm install
```

Comment:
why are we installing npm

Reply:
On one hand, pyright is a node.js package. On the other hand, we're installing it via pip and I believe that it is independent of the system node.js/npm because it depends on [nodeenv](https://pypi.org/project/nodeenv/).